README.md

```diff
@@ -70,6 +70,17 @@ except NormalizationError as e:
     #   Other errors might be found even after applying this suggestion.
 ```
 
+Normalize a name by removing disallowed characters:
+
+```python
+# input name with disallowed zero width joiner
+# str -> str
+ens_force_normalize('Ni‍ck.ETH')
+# 'nick.eth'
+# ZWJ removed, no error raised
+# note: might still raise NormalizationError for certain names
```

Comment:
> Should we use "normalize" here?

Change the function name?

Reply:
Yes, so it won't be confused with standard normalization.

Reply:
Suggestions?

I like force_normalize because it is an extension of normalize (normalization is still performed but there are additional transformations applied - removing disallowed characters).

I don't like healing because it implies a "peaceful" operation that fixes a name, while the function acts more like a hammer that normalizes everything until there is nothing left.